apps/meteor/app/slashcommands-archiveroom/server/server.ts

```diff
@@ -1,71 +1,72 @@
 import { Meteor } from 'meteor/meteor';
 import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
-import { IMessage } from '@rocket.chat/core-typings';
 
 import { Rooms, Messages } from '../../models/server';
 import { slashCommands } from '../../utils/lib/slashCommand';
 import { api } from '../../../server/sdk/api';
 import { settings } from '../../settings/server';
 
-function Archive(_command: 'archive', params: string, item: IMessage): void {
-	let channel = params.trim();
+slashCommands.add(
+	'archive',
+	function Archive(_command, params, item): void {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.